src/frontend/src/reducers/userProfile/userSlice.ts

```diff
@@ -34,17 +35,28 @@ export const userProfileSlice = createSlice({
         ) => {
             state.destinations = destinations;
         },
+        setHasViewedStartInstructions: (
+            state,
+            { payload: status }: { payload: boolean }
+        ) => {
+            state.hasViewedStartInstructions = status;
+        },
```

Comment:
Keeping this status in local storage or in the db felt like overkill (especially since user auth tokens don't expire) so I added it to user profile state for now